zealot-old/fetcher/callbacks.ts

```diff
@@ -18,7 +18,7 @@ export class Callbacks {
     this.callbacks = new Map()
   }
 
-  add(name: EventName, cb: (args?: ZealotPayloadValue) => void) {
+  add(name: EventName, cb: (args?: any) => void) {
```

Comment:
Probably not too important here since we are going to be shifting to the rest of the new zealot client soon, but are we planning on adding strictness to no-any as we tighten up our typescript?

Reply:
Yeah that's why I thought it was find to be any here. We are planning to switch to no "implicit" any. An explicit any like this will still be allowed, but not advised.